compose/ui/ui-graphics/src/skikoMain/kotlin/androidx/compose/ui/graphics/SkiaGraphicsContext.skiko.kt

```diff
@@ -19,19 +19,22 @@ package androidx.compose.ui.graphics
 import androidx.compose.runtime.snapshots.SnapshotStateObserver
 import androidx.compose.ui.InternalComposeUiApi
 import androidx.compose.ui.graphics.layer.GraphicsLayer
-import kotlin.js.JsName
 
-/**
- * Create a new [GraphicsContext].
- */
 @InternalComposeUiApi
-@JsName("createGraphicsContext")
-fun GraphicsContext(snapshotObserver: SnapshotStateObserver): GraphicsContext =
-    SkiaGraphicsContext(snapshotObserver)
+class SkiaGraphicsContext() : GraphicsContext {
+    private val snapshotObserver = SnapshotStateObserver { command ->
+        command()
```

Comment:
Note that it can be called from another thread (when caught states changed and `Snapshot.sendApplyNotifications()` is called from another thread).

Let's add a comment:
```
    // can be called from multiple threads
    private fun requestDraw() {
        drawState.value = Unit
    }
```